mysql-test/suite/galera/t/pxc_read_only_schema.test

```diff
@@ -0,0 +1,749 @@
+# by the cluster.
+#
+# === References ===
+#
+# PXC-4137: WSREP applier threads fail to modify read only schemas
+
+--source include/galera_cluster.inc
+CREATE SCHEMA S;
+
+CREATE TABLE S.t1(i INT, j INT, INDEX i1(i));
+ALTER SCHEMA S read only=1;
+SHOW CREATE SCHEMA S;
+ALTER SCHEMA S read only=0;
+DROP SCHEMA S;
+
+--echo #--------------------------------------------------------------------
+--echo # Invalid option in CREATE SCHEMA.
+--echo #--------------------------------------------------------------------
+--error ER_PARSE_ERROR
+CREATE SCHEMA S read only=0;
+
+--echo #--------------------------------------------------------------------
+--echo # ALTER privilege needed to change read only.
+--echo #--------------------------------------------------------------------
+eval CREATE SCHEMA S;
+CREATE USER s_usr;
+--connect (s_con, localhost, s_usr,,)
+--error ER_DBACCESS_DENIED_ERROR
+ALTER SCHEMA S read only=1;
+
+--connection default
+GRANT ALTER ON S.* TO s_usr;
+
+--connection s_con
+ALTER SCHEMA S read only=1;
+ALTER SCHEMA S read only=0;
+
+--connection default
+--disconnect s_con
+DROP USER s_usr;
+DROP SCHEMA S;
+
+--echo #--------------------------------------------------------------------
+--echo # The read only option affects all users.
+--echo #--------------------------------------------------------------------
+CREATE USER con_adm_usr;
+GRANT ALL ON *.* TO con_adm_usr;
+GRANT CONNECTION_ADMIN ON *.* TO con_adm_usr;
+
+CREATE USER super_usr;
+GRANT ALL ON *.* TO super_usr;
+GRANT SUPER ON *.* TO super_usr;
+
+eval CREATE SCHEMA S;
+
+--connect (root_con, localhost, root,,)
+CREATE TABLE S.t(i INT);
+DROP TABLE S.t;
+ALTER SCHEMA S READ ONLY=1;
+--error ER_SCHEMA_READ_ONLY
+CREATE TABLE S.t(i INT);
+ALTER SCHEMA S READ ONLY=0;
+
+--connect (con_adm_con, localhost, con_adm_usr,,)
+CREATE TABLE S.t(i INT);
+DROP TABLE S.t;
+ALTER SCHEMA S READ ONLY=1;
+--error ER_SCHEMA_READ_ONLY
+CREATE TABLE S.t(i INT);
+ALTER SCHEMA S READ ONLY=0;
+
+--connect (super_con, localhost, super_usr,,)
+CREATE TABLE S.t(i INT);
+DROP TABLE S.t;
+ALTER SCHEMA S READ ONLY=1;
+--error ER_SCHEMA_READ_ONLY
+CREATE TABLE S.t(i INT);
+ALTER SCHEMA S READ ONLY=0;
+
+--connection default
+--disconnect root_con
+--disconnect con_adm_con
+--disconnect super_con
+
+DROP USER con_adm_usr;
+DROP USER super_usr;
+
+DROP SCHEMA S;
+
+--echo #--------------------------------------------------------------------
+--echo # Statements in an init file shall not be affected.
+--echo #--------------------------------------------------------------------
+--connection node_2
+CREATE SCHEMA S;
+let INIT_FILE=$MYSQLTEST_VARDIR/tmp/schema_read_only.sql;
+write_file $INIT_FILE;
+  CREATE SCHEMA schema_read_only;
+  ALTER SCHEMA schema_read_only READ ONLY=1;
+  ALTER SCHEMA schema_read_only COLLATE utf8_bin;
+  CREATE TABLE schema_read_only.t(i INT) TABLESPACE innodb_system;
+  DROP SCHEMA schema_read_only;
+  CREATE TABLE S.init_file(i INT);
+EOF
+
+let $wait_counter= 10000;
```

Comment:
please be consistent across the file
`--let` instead of `let`
`--eval` instead of `eval`

Reply:
Most of the test scenarios written in this file are from `./mysql-test/include/schema_read_only.inc` and was adjusted as per PXC's behavior. So, I have left them as-is.